Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/SharedLogic/SearchTry.hs

```diff
@@ -82,6 +82,11 @@ initiateDriverSearchBatch merchant searchReq tripCategory vehicleVariant estOrQu
               driverSelectedFare = Nothing,
               customerExtraFee = customerExtraFee,
               nightShiftCharge = Nothing,
+              customerCancellationDues = customerCancellationDues,
+              nightShiftOverlapChecking = False, -- sending True in rental
+              estimatedDistance = searchReq.estimatedDistance,
```

Comment:
Rename the `distance` param to `actualDistance` then to avoid confusion and make that maybe.